sdk/storage/Azure.Storage.Blobs/tests/BlobTestBase.cs

```diff
@@ -28,11 +30,10 @@ public abstract class BlobTestBase : StorageTestBase
         protected string SecondaryStorageTenantSecondaryHost() =>
             new Uri(TestConfigSecondary.BlobServiceSecondaryEndpoint).Host;
 
-        public BlobTestBase(bool async) : this(async, null) { }
-
-        public BlobTestBase(bool async, RecordedTestMode? mode = null)
+        public BlobTestBase(bool async, BlobClientOptions.ServiceVersion serviceVersion, RecordedTestMode? mode = null)
             : base(async, mode)
         {
+            _serviceVersion = serviceVersion;
```

Comment:
Can we access `serviceVersion` through the `TestContext.CurrentContext.Properties`?